### DISCUSSION TITLE
Organization Oauth2

### DISCUSSION THREAD

**Discussion by melquelima** [community/community#37998] [Other Feature Feedback, Questions, & Ideas]
Hey all,

 i've created a Org and my user is the owner, i created the Oauth2 Token from my account, and unfortunatelly i`m havving the error  message --"Write access to repository not granted."
how do i grant this access?

**Comment by antonblr** [comment]
Hi @melquelima - are you talking about Personal Access Token? Try recreate it with appropriate permissions - see https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token-classic

Specifically, check "(8) Select the scopes you'd like to grant this token." part. I assume you're looking for `write:org`, but that depends on what you're trying to do when this error shows up.
